includes/abstracts/abstract-wc-gateway-ppec.php

```diff
@@ -367,6 +367,26 @@ public function process_refund( $order_id, $amount = null, $reason = '' ) {
 		$txn_data = $old_wc ? get_post_meta( $order_id, '_woo_pp_txnData', true ) : $order->get_meta( '_woo_pp_txnData', true );
 		$order_currency = $old_wc ? $order->order_currency : $order->get_currency();
 
+		
+		// Process a void instead of refund if transaction is authorized only	
```

Comment:
Trailing tab on this line, as well as the blank lines before and after this function.